### PR TITLE
Add sample code in README.md written in Swift.

### PR DESCRIPTION
**Pull request by takecian**
Hi, i added sample code in swift on README.md.
If you like, please merge this.

Regards,
takecian.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +98,79 @@ As of version 0.2.0, you may also swipe a view programmatically:
 [self.swipeToChooseView mdc_swipe:MDCSwipeDirectionLeft];
 ```
 
+#### Swfit
```
Comment (wilzh40) [inline README.md]: Small typo here, also you have to add `MDCSwipeToChooseDelegate` in the class declaration

**Comment by vikmeup** [comment]
@takecian, @wilzh40 can you check if this code compatible with Swift 1.2 ?

**Comment by takecian** [comment]
@vikmeup I checked this code with Swift 1.2(Xcode 6.3 beta (6D543q)). It worked.

**Comment by takecian** [comment]
i also fixed typo..

**Comment by vikmeup** [comment]
Thanks @takecian

**Pull request merged**